packages/core/nuxt-theme-module/theme/components/Notification.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <transition-group tag="div" class="notifications" :name="isMobile ? 'slide' : 'sf-fade'">
+    <SfNotification
+      v-for="notification in notifications"
+      :key="notification.id"
+      :visible="true"
+      :message="notification.message"
+      :action="notification.action && notification.action.text"
+      :type="notification.type"
+      @click:close="notification.dismiss()"
```

Comment:
I suggest put here reference to the function instead of inline call - just `notification.dismiss`